apps/web/src/services/asset-inventory/collector/modules/CollectorItemInfo.vue

```diff
@@ -0,0 +1,236 @@
+<template>
+    <div class="collector-item-info">
+        <div v-if="props.type === CollectorItemInfoType.PLUGIN"
+             class="info-item"
+        >
+            <p class="info-label">
+                {{ props.label }}
+            </p>
+            <div class="plugin">
+                <p-lazy-img :src="props.item.plugin_icon"
+                            width="1.25rem"
+                            height="1.25rem"
+                />
+                <span class="plugin-name">{{ props.item.plugin_name }}</span>
+                <span class="plugin-version">v{{ props.item.plugin_info.version }}</span>
+            </div>
+        </div>
+        <div
+            v-else-if="props.type === CollectorItemInfoType.STATUS"
+            class="info-item"
+        >
+            <p class="info-label">
+                {{ props.label }}
+            </p>
+            <span>No Schedule</span>
+        </div>
+        <div
+            v-else-if="props.type === CollectorItemInfoType.JOBS"
+            class="info-item"
+        >
+            <p class="info-label">
+                {{ props.label }}
+            </p>
+            <div class="jobs-wrapper">
+                <span class="icon-fill-wrapper">
+                    <!-- TODO: Replace with real data-->
+                    <p-i
+                        name="ic_check"
+                        class="check-icon"
+                        height="1rem"
+                        width="1rem"
+                        color="inherit"
+                    />
+                </span>
+                <span class="icon-fill-wrapper">
+                    <p-i
+                        name="ic_check"
+                        class="check-icon"
+                        height="1rem"
+                        width="1rem"
+                        color="inherit"
+                    />
+                </span>
+                <span class="icon-fill-wrapper">
+                    <p-i
+                        name="ic_check"
+                        class="check-icon"
+                        height="1rem"
+                        width="1rem"
+                        color="inherit"
+                    />
+                </span>
+                <span class="icon-fill-wrapper">
+                    <p-i
+                        name="ic_check"
+                        class="check-icon"
+                        height="1rem"
+                        width="1rem"
+                        color="inherit"
+                    />
+                </span>
+                <span class="icon-fill-wrapper">
+                    <p-i
+                        name="ic_check"
+                        class="check-icon"
+                        height="1rem"
+                        width="1rem"
+                        color="inherit"
+                    />
+                </span>
+            </div>
+            <div class="to-history-detail">
+                <span>View history detail</span>
```

Comment:
nitpick: Plz don't forget to replacing it with translation code. And plz add comments for reviewers 🙏